rotkehlchen/validation/validation_schemas.py

```diff
@@ -0,0 +1,80 @@
+from typing import Any, Dict, Union
+
+from marshmallow import Schema, fields, post_load
+
+from rotkehlchen.api.v1.schemas import UnderlyingTokenInfoSchema
+from rotkehlchen.assets.asset import EthereumToken, UnderlyingToken
+from rotkehlchen.assets.typing import AssetType
+
+
+class AssetListSchema(Schema):
```

Comment:
This is not a List. The name is really confusing.

I guess you wanted to differentiate it from `AssetSchema`. So name it `AssetDataSchema`.
```suggestion
class AssetDataSchema(Schema):
```